ui/app-start-intent/src/main/java/de/danoeh/antennapod/ui/appstartintent/MainActivityStarter.java

```diff
@@ -39,4 +41,10 @@ public MainActivityStarter withOpenPlayer() {
         intent.putExtra(EXTRA_OPEN_PLAYER, true);
         return this;
     }
+
+    public MainActivityStarter withOpenFeed(long feedId) {
+        intent.putExtra(EXTRA_FEED_ID, feedId);
+        intent.putExtra(EXTRA_STARTED_FROM_SEARCH, true);
```

Comment:
Please separate these two options and name it something like "add to backstack" instead of "started from search" because that's no longer what is happening there.

Reply:
I wasn't sure what you mean. I added a new extra in the main activity to differentiate from the "started from search" extra.

Reply:
Okay, please now also make two methods out of this. One for each extra. Currently, it is not clear when using the method that it automatically adds more than one extra.

Reply:
Oh, I see. Added a withAddToBackStack method and moved that extra there.